CHANGELOG/v1.18.0.md

```diff
@@ -0,0 +1,163 @@
+- Overhaul Quickstart UX using AzureASOManagedCluster ([#5319](https://github.com/kubernetes-sigs/cluster-api-provider-azure/pull/5319), [@willie-yao](https://github.com/willie-yao))
+
+### Failing Test
+
+- Bump CAPI to v1.8.8 ([#5371](https://github.com/kubernetes-sigs/cluster-api-provider-azure/pull/5371), [@willie-yao](https://github.com/willie-yao))
+- Custom build templates to use curl when az login fails ([#5109](https://github.com/kubernetes-sigs/cluster-api-provider-azure/pull/5109), [@nawazkh](https://github.com/nawazkh))
+- Default to Marketplace images for old Kubernetes versions ([#5266](https://github.com/kubernetes-sigs/cluster-api-provider-azure/pull/5266), [@mboersma](https://github.com/mboersma))
+
+### Bug or Regression
+
+- Refactor(manager): set machinepool true as default ([#5146](https://github.com/kubernetes-sigs/cluster-api-provider-azure/pull/5146), [@KOREAN139](https://github.com/KOREAN139))
+- AKS extensions removed from an AzureManagedControlPlane's `spec.extensions` will now be deleted. ([#5139](https://github.com/kubernetes-sigs/cluster-api-provider-azure/pull/5139), [@nojnhuh](https://github.com/nojnhuh))
+- Fixed a bug preventing Cluster API from authenticating to ASOAPI Managed Clusters with local accounts disabled. ([#5211](https://github.com/kubernetes-sigs/cluster-api-provider-azure/pull/5211), [@nojnhuh](https://github.com/nojnhuh))
+- Fixed an issue where CAPZ was accumulating memory over time that could never be garbage collected. ([#5284](https://github.com/kubernetes-sigs/cluster-api-provider-azure/pull/5284), [@nojnhuh](https://github.com/nojnhuh))
+- Improves MachinePool scale up/down to avoid unnecessary VMSS VM model udpates ([#5164](https://github.com/kubernetes-sigs/cluster-api-provider-azure/pull/5164), [@mweibel](https://github.com/mweibel))
+- Improves state update handling when running long operations on VMSS ([#5292](https://github.com/kubernetes-sigs/cluster-api-provider-azure/pull/5292), [@mweibel](https://github.com/mweibel))
+- Moves webhook registration behind feature gate flags like controller registration already does. ([#5099](https://github.com/kubernetes-sigs/cluster-api-provider-azure/pull/5099), [@bryan-cox](https://github.com/bryan-cox))
+- Perform nil check on private lb ([#5378](https://github.com/kubernetes-sigs/cluster-api-provider-azure/pull/5378), [@nawazkh](https://github.com/nawazkh))
+
+### Other (Cleanup or Flake)
+
+- Bump CAPI to v1.8.4 ([#5186](https://github.com/kubernetes-sigs/cluster-api-provider-azure/pull/5186), [@mboersma](https://github.com/mboersma))
+- Bump CAPI to v1.8.5 ([#5255](https://github.com/kubernetes-sigs/cluster-api-provider-azure/pull/5255), [@mboersma](https://github.com/mboersma))
+- Bump CAPI to v1.9.3 ([#5316](https://github.com/kubernetes-sigs/cluster-api-provider-azure/pull/5316), [@mboersma](https://github.com/mboersma))
```

Comment:
This should say v1.9.4--maybe I failed to update the commit message?

Reply:
The commit message looks right to me. Not sure what happened here